### ISSUE TITLE
v2.0.0 - db:migrate broken with postgresql - Syntax error

### ISSUE THREAD

**Issue by HorayNarea** [shlinkio/shlink#610]
#### How Shlink is set-up

* Shlink Version: 2.0.0
* PHP Version: whichever is in the Docker image [shlinkio/shlink:latest](https://hub.docker.com/r/shlinkio/shlink)
* How do you serve Shlink: Docker image
* Database engine used: PostgreSQL

#### Summary

when the container starts it tries to migrate the database and encounters a syntax-error:
```
  Migrating up to 20200106215144 from 20191020074522                                                                                                                                                                             
  Migration 20200105165647 failed during Pre-Checks. Error An exception occurred while executing 'UPDATE visit_locations SET latitude = "0" WHERE (latitude="") OR (latitude IS NULL)':                                          
                                                                                                                                                                                                                                 
  SQLSTATE[42601]: Syntax error: 7 ERROR:  zero-length delimited identifier at or near """"                                                                                                                                      
  LINE 1: ...isit_locations SET latitude = "0" WHERE (latitude="") OR (la...                                                                                                                                                     
                                                               ^                                                                                                                                                                 
```

#### Current behavior

Syntax error appears in migration

#### Expected behavior

Migrate successfully and start serving the API

#### How to reproduce

run the latest non-v2 docker-image with `DB_DRIVER=postgres` and upgrade to [shlinkio/shlink:latest](https://hub.docker.com/r/shlinkio/shlink)

**Comment by acelaya** [comment]
Thanks for reporting!

I will check what is wrong with that query.

**Comment by acelaya** [comment]
@HorayNarea It turns out I did several mistakes with the last two database migrations, but I think I sorted them out.

Will have v2.0.1 released soon.

**Comment by acelaya** [comment]
I have already published [v2.0.1](https://github.com/shlinkio/shlink/releases/tag/v2.0.1), but docker images will take some time to build.

Give it a one or two hours.

**Comment by HorayNarea** [comment]
Thanks for the quick resolution! I will try v2.0.1 tomorrow :)
